src/library/Uno.Material/Styles/Controls/Flyout.xaml

```diff
@@ -0,0 +1,528 @@
+
+							<VisualStateGroup x:Name="CheckStates">
+								<VisualState x:Name="Unchecked" />
+
+								<VisualState x:Name="Checked">
+									<VisualState.Setters>
+										<Setter Target="CheckGlyph.Opacity"
+												Value="1" />
+									</VisualState.Setters>
+								</VisualState>
+
+								<VisualState x:Name="UncheckedWithIcon">
+									<VisualState.Setters>
+										<Setter Target="TextBlock.Margin"
+												Value="{ThemeResource MenuFlyoutItemPlaceholderThemeThickness}" />
+										<Setter Target="IconRoot.Visibility"
+												Value="Visible" />
+									</VisualState.Setters>
+								</VisualState>
+
+								<VisualState x:Name="CheckedWithIcon">
+									<VisualState.Setters>
+										<Setter Target="TextBlock.Margin"
+												Value="{ThemeResource MenuFlyoutItemPlaceholderThemeThickness}" />
+										<Setter Target="IconRoot.Visibility"
+												Value="Visible" />
+										<Setter Target="CheckGlyph.Opacity"
+												Value="1" />
+									</VisualState.Setters>
+								</VisualState>
+							</VisualStateGroup>
+
+							<VisualStateGroup x:Name="KeyboardAcceleratorTextVisibility">
+								<VisualState x:Name="KeyboardAcceleratorTextCollapsed" />
+								<VisualState x:Name="KeyboardAcceleratorTextVisible">
+									<VisualState.Setters>
+										<Setter Target="KeyboardAcceleratorTextBlock.Visibility"
+												Value="Visible" />
+									</VisualState.Setters>
+								</VisualState>
+							</VisualStateGroup>
+						</VisualStateManager.VisualStateGroups>
+
+						<Grid x:Name="AnimationRoot">
+							<Grid.ColumnDefinitions>
+								<ColumnDefinition Width="Auto" />
+								<ColumnDefinition Width="*" />
+								<ColumnDefinition Width="38" />
+								<ColumnDefinition Width="Auto"
+												  MinWidth="16" />
+							</Grid.ColumnDefinitions>
+
+							<Path x:Name="CheckGlyph"
+								  Data="{StaticResource CheckGlyphPathStyle}"
+								  Fill="{TemplateBinding Foreground}"
+								  VerticalAlignment="Center"
+								  Stretch="Uniform"
+								  Opacity="0"
+								  Width="16"
+								  Height="16"
+								  win:Margin="16,0"
+								  not_win:Margin="16,4,16,0" />
+
+							<Viewbox x:Name="IconRoot"
+									 Grid.Column="1"
+									 HorizontalAlignment="Left"
+									 VerticalAlignment="Center"
+									 Width="16"
+									 Height="16"
+									 Visibility="Collapsed">
+								<ContentPresenter x:Name="IconContent"
+												  Content="{TemplateBinding Icon}" />
+							</Viewbox>
+
+							<TextBlock x:Name="TextBlock"
+									   Text="{TemplateBinding Text}"
+									   Style="{StaticResource MaterialBody2}"
+									   Foreground="{TemplateBinding Foreground}"
+									   HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
+									   VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
+									   Grid.Column="1" />
+
+							<TextBlock x:Name="KeyboardAcceleratorTextBlock"
```

Comment:
This should be hidden, if the platform doesnt support it.

Reply:
Yeah probably win: on this!